### ISSUE TITLE
Show overall progress (to let a user estimate how much time it may take)

### ISSUE THREAD

**Issue by sanjayankur31** [ironthree/fedora-update-feedback#3]
Is it perhaps possible to say "update x of y" or something to give the user some idea of how far they are in giving feedback? This will help us sort of decide if we want to sit down now to do it or if it's going to take a while and so we should do it later?

**Comment by decathorpe** [comment]
Where would you put this information in the "UI"? I'm not sure whether to include it in the "####" box around the update ID, or above, or below ...

Additionally, it would probably be good to show how many of the "still in queue" updates have already been previously ignored, so, something like `Update 2 / 25 (+5 ignored)` maybe?

**Comment by decathorpe** [comment]
For now, I've implemented this so the numbers are printed right above the input prompt (which is where users are probably looking).

```
################################################################################
##                           FEDORA-2021-c4e724a2ba                           ##
################################################################################

gnupg2-2.3.2-2.fc35
-------------------

Disable ccid driver to avoid clash with pcscd (#2005714)

URL:            https://bodhi.fedoraproject.org/updates/FEDORA-2021-c4e724a2ba
Update type:    bugfix
Submitted:      2021-09-22 13:02:33
Pushed:         2021-09-22 18:46:31
Submitter:      jjelen
Karma:          1
Stable karma:   +3
Unstable karma: -3

Locally installed packages contained in this update:
- gnupg2-2.3.2-2.fc35.x86_64
  Utility for secure communication and data storage
  installed 6 hours and 35 minutes ago
- gnupg2-smime-2.3.2-2.fc35.x86_64
  CMS encryption and signing tool and smart card support for GnuPG
  installed 6 hours and 35 minutes ago

Previous comments:
- besser82 (2021-09-22 21:49:43): +1
  Works great!  LGTM!  =)

This is update 99 out of 100 available updates (including 8 ignored updates).
Action ([S]kip / [i]gnore / [c]omment / [a]bort): 
Skipping.
```

Does this satisfy your need for "are we there yet" info? :)

**Comment by sanjayankur31** [comment]
That's awesome! I'd just have been content with [x/y] before action tbh XD

**Comment by decathorpe** [comment]
Released with version 1.1.0, will end up in Fedora repositories soon.
